examples/regional-dlp/variables.tf

```diff
@@ -14,6 +14,12 @@
  * limitations under the License.
  */
 
+variable "labels" {
+  description = "(Optional) Default label used by Data Warehouse resources."
```

Comment:
s/used by/attached to/

docs seem to like to use the word `attached` to something:
https://cloud.google.com/compute/docs/labeling-resources

Reply:
Also, I don't think the word `default` is needed since it's the only label(s) attached. and default is none, it seems.